### ISSUE TITLE
Костюм-хамелеон

### ISSUE THREAD

**Issue by blackjack22** [FighterX/SOVIET-STATION#14]
Нужно добавить одноименному костюму функцию непосредственно хамелеона. То есть, сливание с окружающими объектами, как у одноименного ящерика. Сделать это очень просто, инфа 100%

"Как???7"
 Достаточно запилить alpha персонажа= не меньше 15 не больше 50 при нажатии на костюме ПКМ и выборе "toggle stealth button"

 "Зачем?7??"

Ну давайте посмотрим аналогичные устройства:
chameleon projector - делает персонажа похожим на окурок или иную вещь. Палева практически нет, если не двигаться. Самый большой минус в том, что перемещаться нельзя.
cloacking device - хренотень, которая делает тебя невидимым. Минусы - не делает невидимым для ХУДов, т.е. очков СБ или медицинского ХУДА. Так как ХУДы есть поголовно у всех овощей, смысла в этом приборе очень мало. Палево: МАКСИМАЛЬНОЕ.

 Как видите, стелс слабореализован, а хотелось бы большего.

 "Эта вещь слишком имба!!!11!"

 Эта хрень не делает вас невидимым. Оно делает вас более незаметным и позволяет тихонько красться по темным коридорам. Тем более, через термалы все равно палево, если персонаж спрятался за стеной, так что не имба.

**Comment by KRLL** [comment]
Ну давайте, что уж. Не думаю, что слишком круто. Можно сделать альфу 100 при беге и 40 при ходьбе.

**Comment by blackjack22** [comment]
Все равно много. Нужно чтобы было чуть-чуть видно. 15 при ходьбе, 50 при беге

**Comment by KRLL** [comment]
Нет, не много. У призрака вообще 127 и его почти не видно. По крайней мере обычно нужно приглядеться, чтобы заметить. И не забывай, что максимум 255, просто для сравнения. А при беге во время любой маскировки будет очень хорошо видно, что какая то хрень невидимая бегает. С альфой 40 вообще почти не видно и не так имба.

**Comment by aleksix** [comment]
При ходьбе-то альфу лучше сделать под 80, иначе медленная машина убийств.
А так, идея фигня. Смысл тогда иметь Chameleon Projector или тот же Cloak? Да и с ХУДами видно будет.

**Comment by KRLL** [comment]
Прятаться в темноте. Только от СБ не скрыться. Я вот подумал, может сделать устройство, блокирующие ХАД очки? Вместе с хамелеоном будет как раз, но тогда больше ничего и не сможешь купить.

**Comment by aleksix** [comment]
Кроме набора джампсьют+блок покупать никто ничего не будет. А что, бегаешь незасекаемый и всех убиваешь.

**Comment by blackjack22** [comment]
НЕ БУДЕТ с худом видно, худ тоже прозрачный

**Comment by blackjack22** [comment]
И засечь как раз можно, если всматриваться и искать
